docs/Manuals/Advanced/map-generator.rst

```diff
@@ -0,0 +1,234 @@
+
+
+Finalizing the Map and Player Placement
+=======================================
+
+This is probably one of the hardest aspects of map generation. Actually getting the map you want as a game
+master, but also ensuring that player placement (or even team placement) is done the way you want is not
+exactly directly mapped. There is a bit of randomness involved.
+
+These options are the last piece to defining a game map:
+
+* ``generator``
+* ``landmass``
+* ``mapseed``
+* ``startpos``
+* ``teamplacement``
+
+As with the ``revealmap`` option, discussed earlier, the ``mapseed`` option is used during testing to keep
+randomness from creeping into your testing. By setting a value, you eliminate the RNG in the server from
+impacting the game map you want to create. If a game master is also playing the game, it is recommended to
+disable (remove or comment out) this option when the game starts so even the game master does not have
+knowledge of the map at game start.
+
+We will get to some recipes in a bit, but before we do that, let us talk about the ``generator`` and
+``startpos`` options. They work in tandem with each other.
+
+First up, ``generator`` has the following configurations:
+
+* ``SCENARIO`` -- This configuration is for Scenario games only. This is a special use case.
+* ``RANDOM`` -- The default. As the name implies, there is a dependency on the built-in Random Number
+  Generator (RNG) in the server. The generator will attempt to create equally spaced, relatively small
+  islands. Player placement will be impacted by the ``landmass`` option. The larger the value the bigger the
+  continents/islands. This option is also impacted by the ``mapsize`` option. Best to use the ``FULLSIZE`` or
+  the ``XYSIZE`` configuration.
+* ``FRACTAL`` -- This is the setting most Longturn games use. This configuration will create earth-like maps.
+  By default, all players are placed on the same continent. The ``landmass`` option can also impact placement.
+* ``ISLAND`` -- Each player is placed on their own island. Each island is similar in size, but not shape.
+* ``FAIR`` -- Every player gets the exact same island.
+* ``FRACTURE`` -- Similar to ``FRACTAL``, however this configuration often places mountains on the coasts.
+
+Now let us discuss ``startpos``, which has the following configurations:
+
+* ``DEFAULT`` -- The default. This configuration uses the ``generator`` configuration to place players.
+* ``SINGLE`` -- One player per island/continent.
+* ``2or3`` -- As the configuration name implies, the ``startpos`` will place 2 or 3 players together on an
+  island/continent.
+* ``ALL`` -- Everyone is placed on the same continent. Make sure you give enough tiles when using this
+  configuration. The ``landmass`` and ``tilesperplayer`` will come in handy.
+* ``VARIABLE`` -- The server will use the RNG to give a bit of randomness to player placement. The size of the
+  continents will be taken into account.
+
+:strong:`Recipe: Large Pangea-like world`
```

Comment:
`aspell` suggests changing Pangea to Pangaea, but looking in different dictionaries suggests that Pangea is a valid alternative spelling. As I am not a native speaker I just leave this note here for your consideration.